platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -57,6 +57,10 @@ RUN pip install urllib3
 RUN pip install requests
 RUN pip install crontab
 
+# Get package to support Dynamic Port Breakout
```

Comment:
s/Get package to support Dynamic Port Breakout/Install dependencies for Dynamic Port Breakout/